lk2nd/device/dts/msm8994/msm8992-lg-bullhead.dts

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	model = "LG Nexus 5X";
+	qcom,msm-id = <251 0>, <252 0>;
```

Comment:
```suggestion
	qcom,msm-id = <QCOM_ID_MSM8992 0>;
```
Let's use the `qcom,ids.h` like all other files around, I also doubt there is a phone with APQ8092 :)

Reply:
True :-) Sorry, I should've checked other examples before adding that, will fix it once I get home